go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/gxthrj/apisix-types v0.1.0
 	github.com/gxthrj/seven v0.1.9
 	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
+	github.com/onsi/ginkgo v1.11.0
```

Comment:
seems wrong. only test cases need this library

Reply:
Since we only use on go module in this project, all dependencies include the tests will be recorded in go.mod.

Reply:
The `go.sum` and `go.mod` in the test can be maintained separately.

![image](https://user-images.githubusercontent.com/6814606/102294035-52f53780-3f83-11eb-879d-92ab5f093691.png)

Reply:
OK.

Reply:
Resolve.